src/tasks/dimensionality_reduction/methods/simlr/script.R

```diff
@@ -0,0 +1,67 @@
+library(anndata)
+library(SIMLR)
+
+## VIASH START
+par <- list(
+  input = "resources_test/dimensionality_reduction/pancreas/dataset.h5ad",
+  output = "output.h5ad", 
+  n_clusters = NULL,
+  n_dim = NA,
+  tuning_param = 10,
+  impute = FALSE, 
+  normalize = FALSE, 
+  cores_ratio = 1
+)
+meta <- list(
+  functionality_name = "simlr"
+)
+## VIASH END
+
+cat("Reading input files\n")
+input <- anndata::read_h5ad(par[["input"]])
```

Comment:
just for future reference in r the items in a list can also be called with `par$input`. Much easier to type.